composer/core/logging/logger.py

```diff
@@ -104,29 +115,58 @@ def __init__(
         self.run_name = run_name
         self._state = state
 
-    def data(self, log_level: Union[str, LogLevel], data: LoggerDataDict) -> None:
+    def data(self, log_level: Union[str, int, LogLevel], data: LoggerDataDict) -> None:
         """Log data to the :attr:`destinations`.
 
         Args:
-            log_level (Union[str, LogLevel]): A :class:`LogLevel`.
+            log_level (str | int | LogLevel): The log level, which can be a name, value, or instance of
+                :class:`LogLevel`.
             data (LoggerDataDict): The data to log.
         """
-        if isinstance(log_level, str):
-            log_level = LogLevel[log_level.upper()]
+        log_level = LogLevel(log_level)
 
         for destination in self.destinations:
             destination.log_data(self._state, log_level, data)
 
+    def file_artifact(
```

Comment:
I'd like to see this function meet a "VERB_NOUN" naming convention. Maybe just `log_artifact`? `file_artifact` is a bit redundant anyways - aren't all artifacts files?

Reply:
Same comment re: https://github.com/mosaicml/composer/pull/698#discussion_r826340994

Reply:
Leaving as-is per discussion in https://github.com/mosaicml/composer/pull/698#discussion_r826340994